elements/pfe-sass/mixins/_custom-properties.scss

```diff
@@ -189,7 +189,7 @@
             @include pfe-set-local(
                 ( BackgroundColor: #{pfe-color(surface--#{$color})} )
             );
-            --theme: #{$theme};
+            --theme: var(--pfe-theme--color--surface--#{$color}--theme, #{$theme});
```

Comment:
👍 Looks great.  Could use `#{pfe-color(surface--#{$color}--theme)}` to capture the same value too as one option but it will render the same.